### DISCUSSION TITLE
release plan for v3?

### DISCUSSION THREAD

**Discussion by dirkmueller** [distribution/distribution#4328] [General]
Is there a plan to cut releases from the main branch? the last available release is quite a while old and a lot of changes have gone into main since then. it would be good to cut a stable release so that consumers can upgrade to that. What is the timeline expectation around that?

**Comment by milosgajdos** [comment]
The short answer is: no, there isn't any coherent plan at the moment.

There are some initiatives alas we have yet to see if they materialize.

**Comment by iseyer** [comment]
Would love to bump this to see if there is any change of status here. 

Would be great to be able to plan changes to harbor (and other downstream consumers) if so.

**Reply by milosgajdos** [reply]
Thanks for the bump, alas nothing has changed. You can use the latest beta release in the meantime.
